vm/src/obj/objslice.rs

```diff
@@ -141,6 +175,12 @@ fn to_index_value(vm: &VirtualMachine, obj: &PyObjectRef) -> PyResult<Option<Big
     }
 }
 
+fn _eq(a: PyObjectRef, b: PyObjectRef, vm: &VirtualMachine) -> PyResult<bool> {
+    let eq = vm._eq(a.clone(), b.clone())?;
+    let value = objbool::boolval(vm, eq)?;
+    Ok(value)
```

Comment:
This pattern is used a lot in the codebase; if you'd like, you could add this as a method (`bool_eq`?) on `VirtualMachine` given that you've already separated it out into its own function here.

Reply:
I think that's a good idea. Fixed vm to have a `bool_eq` function.